src/Node/ElseClauseNode.php

```diff
@@ -14,6 +14,6 @@ class ElseClauseNode extends Node {
     public $elseKeyword;
     /** @var Token */
     public $colon;
-    /**@var StatementNode | StatementNode[] */
+    /** @var StatementNode | StatementNode[] */
```

Comment:
Are spaces between union types allowed? I thought a space separated the type from a description